src/mscorlib/shared/System/Text/UTF8Encoding.cs

```diff
@@ -2496,7 +2498,8 @@ public override byte[] GetPreamble()
             else
                 return Array.Empty<byte>();
         }
-
+        
+        public override ReadOnlySpan<byte> Preamble => _emitUTF8Identifier ? s_preamble : Array.Empty<byte>();
```

Comment:
If somebody inherited from UTF8Encoding and overridden `GetPreamble()` method, the override is not going to be respected for this property. Maybe worth a comment at least - if we consider it acceptable.